sycl/source/detail/graph_impl.cpp

```diff
@@ -71,6 +71,7 @@ bool checkForRequirement(sycl::detail::AccessorImplHost *Req,
   }
   return SuccessorAddedDep;
 }
+
```

Comment:
Would remove this change from the patch, even if it's correct for clang-format it's best practice to touch the minimum number of files possible in a single PR.